externs/browser/w3c_eventsource.js

```diff
@@ -57,17 +57,17 @@ EventSource.prototype.withCredentials;
 /**
  * @const {number}
  */
-EventSource.prototype.CONNECTING = 0;
+EventSource.CONNECTING = 0;
```

Comment:
Looks like https://github.com/google/closure-compiler/pull/3293 already discusses values in externs, so I'll wait for that discussion to be resolved.

Reply:
Ok @lauraharker - I have removed values from this extern to follow discussion in #3293 and changed the PR so that rather than moving constants to symbol that they are instead on both as both are required

Reply:
It should be ready to re-review again.

Reply:
Sounds good, thanks!